cpp/test.py

```diff
@@ -0,0 +1,2 @@
+def harness(p):
+    assert "Hello world++!" in p.readline()
```

Comment:
```suggestion
    assert "Hello World++!" in p.readline()
```

alternatively (personally prefer the fail message in this case, if occure):

```suggestion
    p.expect_exact('Hello World++!')
```